swagger/en/players.yml

```diff
@@ -53,10 +47,6 @@ components:
   schemas:
     player:
       $ref: 'schemas/index.yml#/player'
-    season:
-      $ref: 'schemas/index.yml#/season'
-    gameModeStats:
-      $ref: 'schemas/index.yml#/gameModeStats'
     matchList:
       $ref: 'schemas/index.yml#/matchList'
     
```

Comment:
You don't need to change this, but I recommend that you set your editor to automatically add a newline at the end of a file on save.